templates/python/partial_api_args.mustache

```diff
@@ -0,0 +1,25 @@
+(
+        self,
+        {{#allParams}}
+        {{^vendorExtensions.x-isDomain}}
+        {{paramName}}: {{{vendorExtensions.x-py-typing}}}{{^required}} = None{{/required}},
+        {{/vendorExtensions.x-isDomain}}
+        {{/allParams}}
+        {{#allParams}}
+        {{#vendorExtensions.x-isDomain}}
+        {{paramName}}: {{{vendorExtensions.x-py-typing}}} = "default",
+        {{/vendorExtensions.x-isDomain}}
+        {{/allParams}}
+        _request_timeout: Union[
+            None,
+            Annotated[StrictFloat, Field(gt=0)],
+            Tuple[
+                Annotated[StrictFloat, Field(gt=0)],
+                Annotated[StrictFloat, Field(gt=0)]
+            ]
+        ] = None,
+        _request_auth: Optional[Dict[StrictStr, Any]] = None,
+        _content_type: Optional[StrictStr] = None,
+        _headers: Optional[Dict[StrictStr, Any]] = None,
+        _host_index: Annotated[StrictInt, Field(ge=0, le={{#servers.size}}{{servers.size}}{{/servers.size}}{{^servers.size}}1{{/servers.size}})] = 0,
+    )
```

Comment:
Is this indentation on the last line suppose to be there?

Reply:
Ohhhh, yes!
This is a mustache partial template. It will be templated inside another template, and mustache does not seem to have good whitespace control for that. And you can surely imagine how this plays nicely with python. XD

Reply:
https://github.com/mathialo/bython